python/pyspark/cloudpickle.py

```diff
@@ -194,7 +194,7 @@ def save_function(self, obj, name=None):
         # we'll pickle the actual function object rather than simply saving a
         # reference (as is done in default pickler), via save_function_tuple.
         if islambda(obj) or obj.__code__.co_filename == '<stdin>' or themodule is None:
-            #print("save global", islambda(obj), obj.__code__.co_filename, modname, themodule)
+            # print("save global", islambda(obj), obj.__code__.co_filename, modname, themodule)
```

Comment:
here are the list of ignored files in tox.ini: cloudpickle.py,heapq3.py,shared.py
Look like it would be better to style them anyway. Could I remove them from the exclusion list?